### PR TITLE
HIP-820 Integration of Wallet Connect 2.0 Protocol

### PR DESCRIPTION
**Pull request by bugbytesinc**
Signed-off-by: Jason Fabritz <github@bugbytes.com>

### CODE REVIEW

File: HIP/hip-820.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+---
+hip: 820
+title: Integration of Wallet Connect 2.0 Protocol for Wallets and dApps on Hedera
+author: Jason Fabritz (@bugbytesinc)
+working-group: Jason Fabritz (@bugbytesinc), Milan Wiercx van Rhijn (@MilanWR), Tyler McDonald (@hgraph-io), Ken Anderson (@kenthejr), Tony Camero (@tonycamero), Tyler Cote (@teacoat), Adrian Marin (@AdrianKBL)
```
Comment (hgraphql) [inline HIP/hip-820.md:5]:
```suggestion
working-group: Jason Fabritz (@bugbytesinc), Milan Wiercx van Rhijn (@MilanWR), Tyler McDonald (@hgraph-io), Ken Anderson (@kenthejr), Tony Camero (@tonycamero), Tyler Cote (@teacoat), Adrian Marin (@AdrianKBL), Volodymyr Rozhnov (@Volind) rozhnov.vladimir@gmail.com
```

@bugbytesinc suggesting adding a representative from Blade

Comment (hgraphql) [reply HIP/hip-820.md:5]: whoops, sorry about that. I updated to be just the github handle

File: HIP/hip-820.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
+---
+hip: 820
+title: Integration of Wallet Connect 2.0 Protocol for Wallets and dApps on Hedera
+author: Jason Fabritz (@bugbytesinc)
+working-group: Jason Fabritz (@bugbytesinc), Milan Wiercx van Rhijn (@MilanWR), Tyler McDonald (@hgraph-io), Ken Anderson (@kenthejr), Tony Camero (@tonycamero), Tyler Cote (@teacoat), Adrian Marin (@AdrianKBL), Volodymyr Rozhnov (@Volind)
+type: Standards Track
+category: Application
+needs-council-approval: No
+status: Draft
```
Comment (hgraphql) [inline HIP/hip-820.md:9]:
```suggestion
status: Review
```

File: HIP/hip-820.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+created: 2023-10-05
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/819
+updated: 2023-10-17
```
Comment (hgraphql) [inline HIP/hip-820.md:12]:
```suggestion
updated: 2023-10-31
```

**Comment by hgraphql** [comment]
hey @mgarbs, there's a little confusion surrounding merging the PR. Our intended status is "review". Should we merge as an in-review HIP or should we leave as an open PR?

thanks for clarifying the process!

**Review by mgarbs — APPROVED** [review]

**Pull request merged**